yoti-sdk-api/src/main/java/com/yoti/api/client/ReceiptDetails.java

```diff
@@ -0,0 +1,121 @@
+package com.yoti.api.client;
+
+import java.util.Date;
+import java.util.Optional;
+
+public final class ReceiptDetails {
+
+    private final String id;
+    private final String sessionId;
+    private final String rememberMeId;
+    private final String parentRememberMeId;
+    private final ApplicationProfile profile;
+    private final HumanProfile otherPartyProfile;
+    private final Date date;
```

Comment:
Why use `Date`? `OffsetDateTime` should be used instead. The only reason to use `Date` would be if we wanted Java 7 compatibility